spec/ruby/library/abbrev/abbrev_spec.rb

```diff
@@ -1,31 +1,34 @@
 require_relative '../../spec_helper'
-require 'abbrev'
 
-#test both Abbrev.abbrev and Array#abbrev in
-#the same manner, as they're more or less aliases
-#of one another
+ruby_version_is ""..."3.3" do
+  require 'abbrev'
```

Comment:
Maybe we could keep such specs for 3.3 since these bundled gems should work with `require` and not need a Gemfile?

Reply:
`make test-spec` is invoked without bundled gems. We can't load bundled gems on `ruby/spec` now. 

We should move `spec/ruby/library/*` files to each `ruby/*` repository in the future.

Reply:
> `make test-spec` is invoked without bundled gems.

Mmh, IMO it's always a hack to run test without `make install` first.
Maybe `make test-spec` could skip bundled gems if they are not present?

> We should move `spec/ruby/library/*` files to each `ruby/*` repository in the future.

Probably. I would need help with that, it sounds like a lot of work. And it has the big risk such specs are no longer run by Ruby implementations in CI.

Reply:
Actually, why doesn't just `make` extracts bundled gems? Isn't that fast enough if "cached by Makefile"?
Or maybe `make test-spec` could depend on some Makefile target that does it.
It seems much better to me.